packages/remix-testing/create-remix-stub.tsx

```diff
@@ -96,14 +96,14 @@ export function createRemixStub(
     initialEntries,
     initialIndex,
     hydrationData,
-    remixConfigFuture,
+    future,
   }: RemixStubProps) {
     let routerRef = React.useRef<Router>();
     let remixContextRef = React.useRef<RemixContextObject>();
 
     if (routerRef.current == null) {
       remixContextRef.current = {
-        future: { ...remixConfigFuture },
+        future: { ...future },
```

Comment:
```suggestion
        future,
```

unless you need a copy ? Oo

Reply:
tbh I'm not sure if we need the clone offhand but didn't want to make a change like that at this stage right when we're stabilizing 😬 .  Could definitely be a follow up if we don't need it though